cfgov/v1/wagtail_hooks.py

```diff
@@ -9,6 +9,7 @@
 @hooks.register('after_create_page')
 @hooks.register('after_edit_page')
 def share_the_page(request, page):
```

Comment:
if you revert back to the old code you'll see when it enters this function during a publish:
`print type(page)` 
returns:
`<class 'wagtail.wagtailcore.models.Page'>`